frontend/src/components/cfp-form/index.tsx

```diff
@@ -272,14 +273,15 @@ export const CfpForm: React.SFC<Props> = ({
               }
               errors={getErrors("validationElevatorPitch")}
             >
-              <Textarea
+              <LimitedTextarea
                 sx={{
                   resize: "vertical",
                   minHeight: 340,
                 }}
                 {...textarea("elevatorPitch")}
                 maxLength={300}
                 rows={6}
+                limit={300}
```

Comment:
do we have other places where we have a limit of chars? might be good to add it everywhere we have limits